packages/eslint-plugin-pf-codemods/test/rules/v5/remove-removeFindDomNode.js

```diff
@@ -0,0 +1,204 @@
+const ruleTester = require('../../ruletester');
+const rule = require('../../../lib/rules/v5/remove-removeFindDomNode');
+
+ruleTester.run("remove-removeFindDomNode", rule, {
+  valid: [
+    {
+      code: `import { ApplicationLauncher } from '@patternfly/react-core'; <ApplicationLauncher />`,
+    },
+    {
+      // No @patternfly/react-core import
+      code: `<ApplicationLauncher />`,
```

Comment:
The "no pf import" tests actually are to test that non-pf components are not impacted, so the property should be in there.

Reply:
Also, should the readme be updated?

Reply:
Yep, I figured that distinction out a bit later as I was testing another codemod haha. I'll update the test and also the readme.